README.md

```diff
@@ -72,87 +68,70 @@ export default () => {
 | requestStatus: `'idle'` \| `'fetching'` \| `'resolved'` \| `'rejected'` \| `'interaction'` | The status of the http request to the `url` parameter.<ul><li> `'idle'` before the request is triggered</li><li> `'fetching'` after the request is triggered and the before we get a response.</li><li> `'resolved'` after getting a successful response. Only if it contains a HAL resource.</li><li> `'rejected'` after getting an error response. Or if response doesn't contain a HAL resource.</li><li> `'interaction'` during the execution of an interaction handler.</li></ul>                                                                                                                                                                                              |
 | requestError: `string`                                                                     | The error message in case the request gets rejected. It will be `null` before getting the response or if the response is successful and contains a HAL resource.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | resourceInteractions: `Object`                                                             | This is an object containing as many entries as interactions (\_options.links) are available in the HAL resource. Each entry has the rel of the interaction as a key, and is a function that you can execute passing a payload as a parameter. Executing one of these functions will: <ul><li>Make the http request associated to the given interaction.</li><li>Change the `requestStatus` to `'interaction'`, and then back to `'resolved'` (even when the request fails).</li><li>Update the `resource` if the request responds with a new representation of the same resource.</li><li>Return a promise, so that you can handle the resolution or rejection manually.</li></ul> |
```

Comment:
What means the word `rel`?

Reply:
Also, I think would be better to put `_options.links` between ``.

Reply:
Done!